x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/external/request/HttpRequestTests.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.external.request;
+
+import org.apache.http.client.methods.HttpRequestBase;
+import org.elasticsearch.test.ESTestCase;
+
+import static org.mockito.Mockito.mock;
+
+public class HttpRequestTests extends ESTestCase {
```

Comment:
Since this class doesn't contain any tests, perhaps it should be an abstract class with a name like `HttpRequestTestBase` or something? Or are we expecting that something from ESTestCase is running here?

Reply:
Yeah I just added `ESTestCase` figuring we might add some unit tests later but I doubt that'll happen for `HttpRequest` since it's a simple record. I'll remove it.